src/lib/storage/table.cpp

```diff
@@ -18,56 +18,102 @@
 namespace opossum {
 
 Table::Table(const ChunkOffset target_chunk_size) {
```

Comment:
```suggestion
Table::Table(const ChunkOffset target_chunk_size) : _chunk_size{target_chunk_size} {
```
You could make use of member initialization like this as well.